qiita_pet/handlers/api_proxy/studies.py

```diff
@@ -173,7 +173,8 @@ def study_prep_get_req(study_id, user_id):
                 info['start_artifact_id'] = start_artifact.id
                 info['youngest_artifact'] = '%s - %s' % (
                     youngest_artifact.name, youngest_artifact.artifact_type)
-                info['ebi_experiment'] = bool(prep.ebi_experiment_accessions)
+                info['ebi_experiment'] = bool(
```

Comment:
Seems more concise, but not necessary. This is good to be merged pending @josenavas' comment.